fe-public/src/components/DarkMode.astro

```diff
@@ -0,0 +1,50 @@
+
+<select name="themeSwitch" id="themeSwitch">
+  <option value="dark">DARK MODE</option>
+  <option value="light">LIGHT MODE</option>
+</select>
```

Comment:
Maybe we could change this to just a single button that toggles `on-click`? Or maybe add some nerdy way of calling dark/light mode 😆